MekHQ/src/mekhq/gui/dialog/MedicalViewDialog.java

```diff
@@ -326,30 +324,30 @@ private JPanel fillDoll(JPanel panel, Campaign c, Person p) {
                         default:
                             col = Color.WHITE;
                             break;
-                        
+
                     }
                     doll.setLocColor(bl, col);
                 }
             });
-        if(isGMMode()) {
+        if(campaign.isGM()) {
```

Comment:
Spacing

Reply:
This is at the appropriate indentation level.

Reply:
Sorry, space between the if and the (

Reply:
The file (for whatever reason) omits a space between the `if` and the `(`. To avoid cluttering the PR with whitespace only changes I maintained the original style.